.env.hb.example

```
@@ -0,0 +1,6 @@
+# Reference: https://hyperbeam.ar.io/run/configuring-your-machine.html
+HB_PORT=
+HB_STORE=
+HB_PRINT='<some setting>'
+HB_CONFIG_LOCATION='./config.flat'
+HB_KEY='./wallets/<some-key.json>
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)